articles/aks/istio-native-sidecar.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Enable native sidecar mode for Istio-based service mesh addon in Azure Kubernetes Service (AKS) (preview)
+description: Enable native sidecar mode for Istio-based service mesh addon in Azure Kubernetes Service (AKS) (preview)
+ms.topic: conceptual
+ms.date: 05/07/2024
+ms.author: fuyuanbie
+---
+
+# Enable native sidecar mode for Istio-based service mesh addon in Azure Kubernetes Service (AKS) (preview)
+
```

Comment:
Even though the Istio blog already covers this, should we provide a brief description of what native sidecar mode is and benefits of using it with Istio?